roles/ovb/bmc/files/setup_bmc_node.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
```

Comment:
I fail to see where this file is being used. There's one other mention and it's commented out